cocos/2d/CCScene.cpp

```diff
@@ -254,8 +254,6 @@ void Scene::render(Renderer* renderer, const Mat4& eyeTransform, const Mat4* eye
         _physics3DWorld->debugDraw(renderer);
         renderer->render();
 
-        physics3dDebugCamera->restore();
-
```

Comment:
why remote it?

Reply:
the method no longer exists and not referenced